sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/domain/AbstractEntity.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.domain;
+
+import org.springframework.data.annotation.Id;
+
+public abstract class AbstractEntity {
```

Comment:
Just curious, does this need to be abstract?

Reply:
Probably not, that is what the user had in the issue so I was testing for their exact case. I can change it!

Reply:
Done.